src/helpers/childrenAsArray.ts

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react';
+
+export default function childrenAsArray(children: ReactNode): ReactNode[] {
+  if (Array.isArray(children)) {
+    return children;
+  } else if (typeof children !== 'undefined') {
+    return [children];
+  } else {
```

Comment:
you don't need the `else`, 'cause you are already returning if it fits in any other flow. :)